modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -823,132 +826,130 @@ public void testAddObjectEntry() throws Exception {
 	@Test
 	public void testAddObjectEntryAccountEntryRestriction() throws Exception {
 
-		// Account entry scope
+		// Account entry restricted scope
 
 		AccountEntry accountEntry1 = _addAccountEntry();
 		_user = _addUser();
 
-		_accountEntryUserRelLocalService.addAccountEntryUserRel(
-			accountEntry1.getAccountEntryId(), _user.getUserId());
+		_assignAccountEntryRole(accountEntry1, _buyerRole, _user);
+
+		_addResourcePermission(ObjectActionKeys.ADD_OBJECT_ENTRY, _buyerRole);
+
+		AccountEntry accountEntry2 = _addAccountEntry();
 
 		AssertUtils.assertFailure(
+			ObjectDefinitionAccountEntryRestrictedException.class,
 			StringBundler.concat(
 				"User ", _user.getUserId(),
-				" must have ADD_OBJECT_ENTRY permission for ",
-				_objectDefinition3.getResourceName(), StringPool.SPACE),
-			() -> _addObjectEntry(accountEntry1));
+				" does not have access to account entry ",
+				accountEntry2.getAccountEntryId()),
+			() -> _addObjectEntry(accountEntry2));
 
-		_addResourcePermission("ADD_OBJECT_ENTRY", _buyerRole);
+		// Organization account entry restricted scope
```

Comment:
@murilostodolni can we rename it to `Account entry restricted scope with organization`?